web/static/js/components/css_modules/prime_directive.css

```diff
@@ -0,0 +1,8 @@
+.wrapper {
```

Comment:
hey @qlaire, I think these styles can be avoided by using classes provided by semantic ui. as it provides means of centering content, limiting widths, et cetera. @tnewell5 can help you out, as we used semantic ui on the forecast manager project